packages/web-app-files/src/components/PublicLinksSidebar/LinkActions.vue

```diff
@@ -9,7 +9,7 @@
         class="oc-files-file-link-edit oc-mr-xs"
         @click="editLink"
       >
-        <oc-icon name="edit" />
+        <oc-icon id="editButtonLabel" name="edit" />
```

Comment:
Id is not unique, since this component exists once per link row. You need to add a unique qualifier to the id dynamically (e.g. the share id).